src/frontend/utils/tests/render.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  act,
+  render,
+  RenderOptions as TestingLibraryRenderOptions,
+  screen,
+} from '@testing-library/react';
+import { Grommet, ResponsiveContext, ThemeType } from 'grommet';
+import MatchMediaMock from 'jest-matchmedia-mock';
+import React, { CSSProperties, ReactElement } from 'react';
+import toast, { Toast, Toaster, useToaster } from 'react-hot-toast';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import { BreadCrumbsProvider } from 'components/BreadCrumbs';
+import { wrapInIntlProvider } from 'utils/tests/intl';
+import { wrapInRouter } from 'utils/tests/router';
+import { GlobalStyles } from 'utils/theme/baseStyles';
+
+//  ------- interfaces -------
+interface IntlOptions {
+  locale?: string;
+}
+interface RouterOptions {
+  routes?: { path: string; render: ({ match }: any) => JSX.Element }[];
+  componentPath?: string;
+  header?: React.ReactElement;
+  history?: string[];
+}
+interface QueryOptions {
+  client: QueryClient;
+}
+interface GrommetOptions {
+  theme?: ThemeType;
+  style?: CSSProperties;
+  responsiveSize?: string;
+}
+
+/**
+ * Options to configure the render of a component for a test
+ *
+ * @property testingLibraryOptions options provided by react-testing-library to the render method
+ * @property intlOptions options to configure i18n context
+ * @property routerOptions options to configure router and routes in the test
+ * @property queryOptions options to configure a custom client used by react-query for a test
+ * @property grommetOptions options to configure grommet theme, responsive and add style to the mounted base div
+ */
+interface RenderOptions {
+  testingLibraryOptions: TestingLibraryRenderOptions;
+  intlOptions: IntlOptions;
+  routerOptions: RouterOptions;
+  queryOptions: QueryOptions;
+  grommetOptions: GrommetOptions;
```

Comment:
would be so nice to have it alphabetically ordered, just to say something :-)

Reply:
I changed it but imo we should either find a way to automate this or totally ignore it